packages.yml

```
@@ -1,5 +1,10 @@
 packages:
-- package: fivetran/shopify
-  version: [">=0.10.0", "<0.11.0"]
+# - package: fivetran/shopify
+#   version: [">=0.11.0", "<0.14.0"]
+
+- git: https://github.com/fivetran/dbt_shopify.git
+  revision: feature/standardized-line-item-model
+  warn-unpinned: false
```

```suggestion
- package: fivetran/shopify
  version: [">=0.11.0", "<0.14.0"]
```
I would remove the branch before merging since we know it won't be released when this is released.